composer.json

```diff
@@ -24,15 +24,16 @@
         }
     ],
     "require":     {
-        "php":       "^5.4|^7",
+        "php": "^5.4 || ^7",
         "psr/cache": "~1.0",
         "cache/tag-interop": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.0|^5.0",
+        "phpunit/phpunit": "^4.0 || ^5.0",
         "cache/cache": "dev-master",
         "symfony/cache": "^3.1",
         "madewithlove/illuminate-psr-cache-bridge": "^1.0",
+        "mockery/mockery": "^0.9.9",
```

Comment:
https://travis-ci.org/php-cache/integration-tests/jobs/200021430 
and scroll down